src/ring_theory/ideal/operations.lean

```diff
@@ -407,28 +384,23 @@ lemma span_singleton_mul_span_singleton (r s : R) :
   span {r} * span {s} = (span {r * s} : ideal R) :=
 by { unfold span, rw [submodule.span_mul_span, set.singleton_mul_singleton], }
 
-lemma span_singleton_pow (s : R) (n : ℕ):
-  span {s} ^ n = (span {s ^ n} : ideal R) :=
+lemma span_singleton_pow (s : R) (n : ℕ) : span {s} ^ n = (span {s ^ n} : ideal R) :=
```

Comment:
Again, the type is more interesting than everything before it, so I'd give it its own line:
```suggestion
lemma span_singleton_pow (s : R) (n : ℕ) :
  span {s} ^ n = (span {s ^ n} : ideal R) :=
```